test/remote/gateways/remote_paystation_test.rb

```diff
@@ -103,6 +104,15 @@ def test_invalid_login
     assert_failure response
     assert_nil response.authorization
   end
+
+  def test_store_with_hmac
+    time = Time.now.to_i
+    assert response = @hmac_gateway.store(@credit_card, @options.merge(:order_id => get_uid, :token => "justatest#{time}"))
```

Comment:
Don't you need to pass in the `:hmac_authentication_key` param to actually test this?